lib/base-chips/field-types/text.js

```diff
@@ -4,12 +4,10 @@ const escape = require("escape-html");
 
 module.exports = {
 	name: "text",
-	full_text_search_enabled: function(params){
-		if (params && params.include_in_search){
-			return true;
-		} else if (params && params.max_length > 200){
-			return true;
-		} else {
+	has_index: function(params){
```

Comment:
PS you might want to read the corresponding changes in the sealious-datastore-dbs-common-parts https://github.com/sealcode/sealious-datastore-dbs-common-part/commit/649b7446f9f28f535339390747ad8b481dcba8ab